Gemfile.lock

```diff
@@ -184,25 +184,25 @@ GEM
     turbolinks (5.2.1)
       turbolinks-source (~> 5.2)
     turbolinks-source (5.2.0)
-    tzinfo (1.2.6)
+    tzinfo (1.2.7)
       thread_safe (~> 0.1)
     uglifier (4.2.0)
       execjs (>= 0.3.0, < 3)
-    web-console (4.0.1)
+    web-console (4.0.4)
       actionview (>= 6.0.0)
       activemodel (>= 6.0.0)
       bindex (>= 0.4.0)
       railties (>= 6.0.0)
-    webdrivers (4.2.0)
+    webdrivers (4.4.1)
```

Comment:
### [webdrivers](https://github.com/titusfortner/webdrivers)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 4.2.0 → 4.4.1

[_[change-log](https://github.com/titusfortner/webdrivers/blob/master/CHANGELOG.md), [source-code](https://github.com/titusfortner/webdrivers/tree/v4.4.1)_]





<details>
<summary>Commits</summary>

A change of **25** commits. See the full changes on [the compare page](https://github.com/titusfortner/webdrivers/compare/v4.2.0...v4.4.1).

These are the first 10 commits:
- (b43e476) [Update rubocop config as per v0.79.](https://github.com/titusfortner/webdrivers/commit/b43e476513ced0a683be6625f53749f92971ab4a)
- (be78e5d) [Add binary path for Edge stable on Windows.](https://github.com/titusfortner/webdrivers/commit/be78e5d9aeccf1044a790b7c0a467d93cad0117c)
- (3ac32bc) [Add project metadata to the gemspec (#165)](https://github.com/titusfortner/webdrivers/commit/3ac32bcd59db252f68a8a562eb4d2997f61e7c05)
- (77057bf) [Require Ruby >= 2.4.](https://github.com/titusfortner/webdrivers/commit/77057bf20a62455a6f5eee1672c3ddae33c28c93)
- (64a6eec) [Target latest available ruby/jruby versions on TravisCI and …](https://github.com/titusfortner/webdrivers/commit/64a6eec2ebccca814ac6279f1ea6af1d59e9cc11)
- (923c5f1) [Install Microsoft Edge stable for CI jobs.](https://github.com/titusfortner/webdrivers/commit/923c5f19d0e734deef4ee2a40cd599c6f4574b2d)
- (2a557d4) [Update CHANGELOG.](https://github.com/titusfortner/webdrivers/commit/2a557d463e10db8dc7af7babd013b1fb383c1cb2)
- (411c663) [Bump to v4.3.0](https://github.com/titusfortner/webdrivers/commit/411c663945b1e9780e181b7edb4809d2f2a17ff1)
- (e559e0a) [Update Travis configs:](https://github.com/titusfortner/webdrivers/commit/e559e0acb3fc19399df1f6d595f0313bc6ddf651)
- (a50315e) [Update Appveyor configs:](https://github.com/titusfortner/webdrivers/commit/a50315e19845d5de3c1a9548513985346e48ddbd)


</details>